docs/dashboards/scheduled-report.md

```diff
@@ -20,45 +20,74 @@ If a .pdf and/or .png export fails, an error email will be sent to your email ID
 To schedule a dashboard email report, do the following:
 
 1. Navigate to the dashboard for which you want to create a schedule.
-1. Select or enter a time range for the dashboard.
 1. If the dashboard supports template variables, select values, if desired.
-1. Click the kebab menu and select **Create Schedule Report**. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/create_scheduled_report.png')} alt="Create Schedule Report option selected on the dropdown menu" width="200"/>
-    :::info
-    The **Create Schedule Report** option appears only if you have edit permissions for the dashboard. You have edit permissions for all dashboards you create, and any dashboards have been [shared](/docs/dashboards/share-dashboard-new/) with you with edit permissions granted.
```

Comment:
I think keeping some version of this would be helpful as it is still applicable.

Reply:
I retained the same info section.